cli/commands/start/binance-smart-chain.js

```diff
@@ -0,0 +1,17 @@
+// Binance Smart chain configurations to connect with testnet
+class StartBinanceSmartChainNodeCommand {
+  static async execute ({ ethNodeUrl, ethMasterSk, nearClientValidateHeader }) {
+    if (nearClientValidateHeader !== 'true' && nearClientValidateHeader !== 'false') {
+      nearClientValidateHeader = 'true'
+    }
+
+    return {
+      ethNodeUrl: ethNodeUrl !== '' ? ethNodeUrl : 'https://data-seed-prebsc-1-s1.binance.org:8545',
+      ethMasterSk: ethMasterSk !== '' ? ethMasterSk : '0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501200',
```

Comment:
Where is this secret key coming from? is this a good default? should we leave this empty and force the user to set the secret key.

Reply:
The SK is the same SK used for testing with ganache. This SK is used in many places not only for bsc but also for ethhash.